fbpic/lpa_utils/laser/laser_profiles.py

```diff
@@ -835,3 +837,175 @@ def E_field( self, x, y, z, t ):
         Ey = self.E0y * profile
 
         return( Ex.real, Ey.real )
+
+class FromLasyFileLaser( LaserProfile ):
+    """Class that emits a laser from a lasy file"""
+
+    def __init__(self, filename, t_start=0.):
+        """
+        Define a laser whose profile is determined by a
+        `lasy <https://lasydoc.readthedocs.io/en/latest/>`_ file.
+
+        Parameters
+        ----------
+        filename: string
+            The path to the ``lasy`` file.
+
+        t_start: float (in seconds), optional
+            Physical time (in the simulation), at which the laser will start being
+            emitted. This can be used in order to introduce a time delay that was
+            not originally present in the ``lasy`` file.
```

Comment:
```suggestion
            Physical time (in the simulation), at which the laser will start being
            emitted. This can be used in order to introduce a time delay that was
            not originally present in the ``lasy`` file. Note that the beginning of 
            the time axis in the ``lasy`` file is forced to be zero, when initializing
            the laser with FBPIC (irrespective of the metadata for `tmin` that is 
            actually present in the ``lasy`` file.)
```